docs/docs/cli.md

```diff
@@ -245,7 +245,7 @@ poetry add "git+https://github.com/pallets/flask.git@1.1.1[dotenv,dev]"
 * `--path`: The path to a dependency.
 * `--optional` : Add as an optional dependency.
 * `--dry-run` : Outputs the operations but will not execute anything (implicitly enables --verbose).
-
+* `--allow-prereleases`: Install a pre-release version if present for the library. Example: poetry add --allow-prereleases=true
```

Comment:
```suggestion
* `--allow-prereleases`: Accept prereleases.
```

This will actually match the output of `poetry help add`.

Also, the example is not correct. `--allow-preleases` is a flag and, as such, does not accept a value.

I think we can leave out the example.